routes/blogRouter.js

```diff
@@ -3,5 +3,7 @@ const blogRouter = express.Router();
 const blogController = require("../controllers/blogController");
 
 blogRouter.get("/", blogController.blogs_get);
+blogRouter.get("/blog/clicked", blogController.blogs_single_get);
```

Comment:
You'll want to remove "/blog" from this as that's already defined in app.js where we use the blogRouter.